public/app/plugins/panel/canvas/module.tsx

```diff
@@ -58,6 +58,9 @@ export const plugin = new PanelPlugin<Options>(CanvasPanel)
           icon: true,
         },
       },
+      [FieldConfigProperty.Links]: {
+        hideFromDefaults: true,
+      },
```

Comment:
just for context for other reviewers, these changes hide the default data links editor for the canvas plugin